pkg/render/node.go

```diff
@@ -285,6 +306,36 @@ func (c *nodeComponent) nodeRoleBinding() *rbacv1.ClusterRoleBinding {
 	return crb
 }
 
+// cniPluginRoleBinding creates a rolebinding giving the node service account the required permissions to operate.
```

Comment:
Looks like a few of these comments have copy/paste errors - "the node service account"